feg/cloud/go/protos/s6a_proxy.pb.go

```diff
@@ -422,6 +422,8 @@ type AuthenticationInformationAnswer_EUTRANVector struct {
 	Xres                 []byte   `protobuf:"bytes,2,opt,name=xres,proto3" json:"xres,omitempty"`
 	Autn                 []byte   `protobuf:"bytes,3,opt,name=autn,proto3" json:"autn,omitempty"`
 	Kasme                []byte   `protobuf:"bytes,4,opt,name=kasme,proto3" json:"kasme,omitempty"`
+	Ck                   []byte   `protobuf:"bytes,5,opt,name=ck,proto3" json:"ck,omitempty"`
```

Comment:
once you eliminate the changes on s6a_proxy.proto, this file will not be generated and needed.

Reply:
done, regenerated the above file